haas/cli.py

```diff
@@ -247,17 +247,31 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+
+    # Check if config contains usable sleep_time
+    if cfg.has_option('network-daemon', 'sleep_time'):
```

Comment:
Did you check to see if you need a cfg.has_section() first, in case the network-daemon section is missing?